src/std/gtk/README.md

```diff
@@ -0,0 +1,11 @@
+# WIP GTK4 bindings (with example file).
+
+Hello, this what I have been up to for the last month. This is unfinished btw but it works.
+
+Todo:
+ * "Bind" more gui functions for all GUI components
+ * Fix closing the app
+ * fix the event handler and make it more flexible.
+
+Help me please:
+  * How to type cast in CSpydr (lol)
```

Comment:
> How to type cast in CSpydr (lol)
like this: `expr: NewType`, you might need parentheses around it: `(expr: NewType)`